legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/nodes/helpers/platform/DefaultExecutionNodeContext.java

```diff
@@ -82,6 +82,10 @@ else if (clazz.equals(PureDate.class) && value instanceof EngineDate)
             {
                 return (T) ((EngineDate) value).transformToPureDate();
             }
+            else if (clazz.equals(Long.class) && value instanceof String)
```

Comment:
This doesn't seem like the right place for the change. If clazz is Long value should not be string.
Possibly we should fix normalization logic here - https://github.com/finos/legend-engine/blob/master/legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/validation/FunctionParametersParametersValidation.java#L40